### DISCUSSION TITLE
modelserver后端应该配合哪个前端界面启动？

### DISCUSSION THREAD

**Discussion by 136861535** [eclipse-emfcloud/emfcloud#139] [ModelServer]
请问：modelserver后端应该配合哪个前端界面启动？目前我们使用的是coffee-editor的前端界面启动的，但那样有一些问题。

**Comment by eneufeld** [comment]
There is no predefined frontend for the modelserver as it just provides a rest like interface. The modelserver project offers a client integration with java and javascript.
So it is up to you accessing the data.
What is not working with the coffee editor for you?
